### PR TITLE
Release 2.3.0

### PR DESCRIPTION
**Pull request by chris-pettinga**

### CODE REVIEW

File: pii-ner-exclude.txt [hunk 1]
```diff
@@ -1,3 +1,46 @@
...
+htmlcov/
+.cache
+celery
+NER
+parts/
+Installer
+htmlcov/
+.cache
+celery
+NER
```
Comment (nahmed99) [inline pii-ner-exclude.txt:42]: The following are all in there twice. Minor at this point, but will try and see if there are replications in the (actual) code:

parts/
Installer
htmlcov/
.cache
celery
NER

+ a few others in the first 10 lines too.

File: trade_remedies_caseworker/config/db.sqlite3
Comment (nahmed99) [inline trade_remedies_caseworker/config/db.sqlite3:1]: This shouldn't be here?

File: trade_remedies_caseworker/organisations/v2/forms.py [hunk 2]
```diff
@@ -26,3 +27,45 @@ def clean(self):
...
+    )
+
+
+class ConfirmNotDuplicateForm(ValidationForm):
+    ...
+
+
+class CancelMergeForm(ValidationForm):
+    ...
+
```
Comment (nahmed99) [inline trade_remedies_caseworker/organisations/v2/forms.py:65]: > class ConfirmNotDuplicateForm(ValidationForm):
>     ...
> class CancelMergeForm(ValidationForm):
>     ...


Note for future - we can use just Validation form in both case above?

**Review by nahmed99 — APPROVED** [review]

File: trade_remedies_caseworker/organisations/v2/forms.py (hunk 2 above)
Comment (chris-pettinga) [reply trade_remedies_caseworker/organisations/v2/forms.py:65]: @nahmed99 good spot

Comment (chris-pettinga) [reply trade_remedies_caseworker/organisations/v2/forms.py:65]: @nahmed99 good spot

**Pull request merged**